Show cursor when clicking and dragging.

veroxzik opened the pull request:
Cursor should always show when clicking and dragging, even if the option to show the cursor is off.

veroxzik committed:
Show cursor when clicking and dragging.

Cursor should always show when clicking and dragging, even if the option to show the cursor is off.

yellowberryHN commented:
This PR causes a crash when double-clicking on the title bar of the window to maximize.

veroxzik committed:
Resolve crash.

Resizing the window could trigger the MouseMove event which would trigger a crash due to an invalid value.

veroxzik commented:
This has been superseded by #7 .
